### PR TITLE
Broken tests

### PR DESCRIPTION
**Pull request by bvitaliyg**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests
+ [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

**Comment by bvitaliyg** [comment]
Record snapshots

File: .github/workflows/record_snapshots.yml [hunk 1]
```diff
@@ -155,7 +156,7 @@ jobs:
               issue_number: context.issue.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
-              body: `### '${{ matrix.variant }}' snapshots were updated. Please review it.
+              body: `'${{ matrix.variant }}' snapshots were updated. Please review it.
```
Comment (zgeor) [inline .github/workflows/record_snapshots.yml:159]: Why not leave this as heading.3 (###)?

Comment (bvitaliyg) [reply .github/workflows/record_snapshots.yml:159]: It'll be 4 comments with large headings. I think it's rather distracting.

Comment (zgeor) [reply .github/workflows/record_snapshots.yml:159]: This creates an inconsistency with
> 'dm' files were changed. 

type of comments. 

Do you plan to ensure the format is the same across both?

Comment (bvitaliyg) [reply .github/workflows/record_snapshots.yml:159]: The comments you're mentioning are warning comments. It signalizes that something went wrong and developer attention is required.

These types of comments signalize that the task ran successfully.

**Review by zgeor — APPROVED** [review]

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**